zulip_bots/setup.py

```diff
@@ -55,7 +55,8 @@
         'requests',  # for bots/link_shortener and bots/jira
         'python-chess[engine,gaviota]',  # for bots/chess
         'wit',  # for bots/witai
-        'apiai'  # for bots/dialogflow
+        'apiai',  # for bots/dialogflow
+        'simple_salesforce'  # for bots/salesforce
```

Comment:
@aero31aero remember how we once had a discussion about where bot dependencies should be included? Did it have to be in `setup.py`, or was it also possible to add it in a `requirements.txt`?